Library/Homebrew/dev-cmd/bump-cask-pr.rb

```diff
@@ -153,7 +153,7 @@ def bump_cask_pr
       end
     end
 
-    if new_hash.present?
+    if new_hash.present? && cask.language.blank?
```

Comment:
Can you add a comment here explaining why `cask.language` needs to be blank here?